src/lib/task_listener.js

```diff
@@ -3,17 +3,18 @@ Primary interface which handles listening for messages and initializing the
 execution of tasks.
 */
 
-var QUEUE_PREFIX = 'worker/v1/';
 
+import TaskQueue from './queueservice';
+import DeviceManager from './devices/device_manager';
 var debug = require('debug')('docker-worker:task-listener');
```

Comment:
Because I forgot to do away with all of them when updating this file :)  This will be fixed